src/renderer/components/FileDirectory.vue

```diff
@@ -69,20 +72,21 @@ export default {
   },
   computed: {
     /**
-     *
+     * maps the files in a directory to make iterating easier, also sets the active state if necessary
+     * @return Object[]
      */
     files() {
       return this.directory.files.map((file) => {
         return {
-          active: this.active === file,
+          active: this.active?.name === file.name,
```

Comment:
I did not know you could do this. Is the "?" acting as a conditional here?

Reply:
Yep! Its a relatively new feature so normally I wouldn't use it but since this is an electron app I can guarantee the version of chrome we're on will support it.

For reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining